src/OpenTelemetry.Exporter.ZPages/ZPagesProcessor.cs

```diff
@@ -43,22 +43,22 @@ public ZPagesProcessor(ZPagesExporter exporter)
         /// <inheritdoc />
         public override void OnStart(Activity activity)
         {
-            if (!ZPagesActivityTracker.ProcessingList.ContainsKey(activity.DisplayName))
+            if (!ZPagesActivityTracker.ProcessingList.ContainsKey(activity?.DisplayName))
```

Comment:
Should this method support null activity? I think we might just want a `Debug.Assert(activity != null)` and then suppress the warning to null check public parameter. ActivityProcessor is hot path.

Reply:
just updated. can u review?